### PR TITLE
Added tooltip and focus ring when address is invalid

### PR DESCRIPTION
**Pull request by helloitsm3**
PR for #679

### CODE REVIEW

File: src/components/MakeWidget/MakeWidget.tsx [hunk 1]
```diff
@@ -152,6 +154,10 @@ const MakeWidget: FC = () => {
     useShouldDepositNativeToken(makerAmount);
   const shouldDepositNativeToken = !!shouldDepositNativeTokenAmount;
   const hasDepositPending = useDepositPending();
+  const inputHasError =
```
Comment (piersss) [inline src/components/MakeWidget/MakeWidget.tsx:157]:
```suggestion
  const addressInputHasError =
```

**Review by piersss — CHANGES_REQUESTED** [review]
1. Error state shows immediately when user enters character. This is not user friendly behaviour.
2. Border of input should be red, not any div's around it.

File: src/components/MakeWidget/MakeWidget.tsx [hunk 2]
```diff
@@ -310,6 +314,18 @@ const MakeWidget: FC = () => {
     }
   };
 
+  const handleTakerAddress = async (value: string) => {
```
Comment (GiuntaLucas) [inline src/components/MakeWidget/MakeWidget.tsx:317]: Same here, i think we should use a method return type.

**Review by GiuntaLucas — CHANGES_REQUESTED** [review]

File: src/components/MakeWidget/MakeWidget.tsx (hunk 2 above)
Comment (helloitsm3) [reply src/components/MakeWidget/MakeWidget.tsx:317]: we no longer need this. I'm removing it

File: src/components/MakeWidget/MakeWidget.styles.tsx [hunk 3]
```diff
@@ -69,3 +70,15 @@ export const StyledInfoSection = styled(InfoSection)`
     margin-top: 1rem;
   }
 `;
+
+export const StyledTooltip = styled(Tooltip)`
+  position: absolute;
+  right: 0;
+  top: -0.7rem;
```
Comment (piersss) [inline src/components/MakeWidget/MakeWidget.styles.tsx:77]:
```suggestion
  top: 0;
  height: 3.5rem;
```

File: src/components/MakeWidget/MakeWidget.tsx [hunk 4]
```diff
@@ -357,11 +360,23 @@ const MakeWidget: FC = () => {
...
-          onInfoButtonClick={toggleShowOrderTypeInfo}
-        />
+        <TooltipContainer>
+          <StyledAddressInput
+            hasError={!!error}
+            value={takerAddress}
+            onChange={(e) => {
+              setTakerAddress(e);
+              dispatch(reset());
+            }}
```
Comment (piersss) [inline src/components/MakeWidget/MakeWidget.tsx:370]:
```
const handleAddressInputChange = (value: string) => {
    setTakerAddress(value);
    if (error?.type === AppErrorType.invalidAddress) {
      dispatch(setError(undefined))
    }
  }
```

File: src/components/SwapInputs/subcomponents/Tooltip/Tooltip.styles.tsx [hunk 5]
```diff
@@ -8,13 +8,14 @@ const triangleSize = 14;
...
-  width: 100%;
   max-width: 17rem;
-  height: 4.5rem;
+  height: 0;
 
   @media ${breakPoints.tabletLandscapeUp} {
     align-items: center;
     top: 0;
+    width: 100%;
+    height: 4.5rem;
```
Comment (piersss) [inline src/components/SwapInputs/subcomponents/Tooltip/Tooltip.styles.tsx:18]: Oh I see that this styling was broken, nice!

**Review by piersss — CHANGES_REQUESTED** [review]
import errors:

```
src/components/MakeWidget/MakeWidget.tsx
  Line 13:10:  'ethers' is defined but never used     @typescript-eslint/no-unused-vars
  Line 56:10:  'ErrorList' is defined but never used  @typescript-eslint/no-unused-vars
```

Input border wrong on focus:
Screenshot 2022-12-14 at 22 12 42

Disable Take button when error.

**Review by piersss — APPROVED** [review]

**Pull request merged**